server/fleet/hosts.go

```diff
@@ -1207,3 +1207,16 @@ func ValidateHostScriptContents(s string) error {
 
 	return nil
 }
+
+type ScriptResult struct {
+	ScriptContents string `json:"script_contents"`
+	ExitCode       *int64 `json:"exit_code"`
+	Output         string `json:"output"`
+	Message        string `json:"message"`
+	HostName       string `json:"host_name"`
```

Comment:
Minor, but for consistency - in our existing APIs we use `hostname` and not `host_name`.

Reply:
Actually, I think this whole struct is now unused?

Reply:
absolutely right, thank you!